src/CLICommand/common_context.ts

```diff
@@ -50,6 +51,24 @@ export class CommonContext {
 		throw new Error('No wallet file neither seed phrase provided!');
 	}
 
+	public async getDriveKey(driveId: DriveID): Promise<Buffer> {
+		const driveKey = this.getParameterValue(DriveKeyParameter);
+		if (driveKey) {
+			return Buffer.from(driveKey);
+		}
```

Comment:
Here @fedellen: if the user provides its drive key, the wallet and password won't be checked